### PR TITLE
[tests-only][full-ci]Add test for search deleted files using content

### PR DESCRIPTION
**Pull request by nabim777**
## Description
This PR adds the API tests for search deleted files using content. The scenarios added in this PR are
- search deleted files using content

## Related Issue
- https://github.com/owncloud/ocis/issues/6606#event-9635806756

## Motivation and Context
- there was no test coverage for api test for search deleted files using content. so, this PR covers the require scenario test case

## How Has This Been Tested?
- test environment:
- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiFullTextSearch/search.feature [hunk 1]
```diff
@@ -25,3 +25,22 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search deleted files using context
```
Comment (saw-jan) [inline tests/acceptance/features/apiFullTextSearch/search.feature:30]:
```suggestion
  Scenario Outline: search deleted files using content
```

File: tests/acceptance/features/apiFullTextSearch/search.feature [hunk 2]
```diff
@@ -25,3 +25,22 @@ Feature: full text search
...
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "file2.txt"
+    And user "Alice" has deleted file "file1.txt"
+    When user "Alice" searches for "Content:hello" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result should contain "1" entries
+    And the search result of user "Alice" should contain only these files:
+      | file2.txt |
+    And the search result of user "Alice" should not contain these entries:
+      | /file1.txt |
```
Comment (grgprarup) [inline tests/acceptance/features/apiFullTextSearch/search.feature:41]:
```suggestion
    And the search result of user "Alice" should contain only these files:
      | file2.txt |
```

File: tests/acceptance/features/apiFullTextSearch/search.feature [hunk 3]
```diff
@@ -25,3 +25,22 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search deleted files using context
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "file2.txt"
+    And user "Alice" has deleted file "file1.txt"
```
Comment (grgprarup) [inline tests/acceptance/features/apiFullTextSearch/search.feature:34]: Refer to https://github.com/owncloud/ocis/pull/6709#discussion_r1266270454

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by SwikritiT — COMMENTED** [review]
LGTM

**Review by SwikritiT — APPROVED** [review]

**Pull request merged**